public/leaderboardinfo.php

```diff
@@ -72,13 +72,17 @@ function ExplainLeaderboardTrigger(string $name, string $triggerDef, array $code
     echo "</div>"; // devbox
 }
 
+$openGraphDescription = "Compete on the $lbTitle leaderboard. ";
```

Comment:
This page doesn't actually let players compete. I recommend something similar to the achievements OpenGraph description:
```
$openGraphDescription = ($lbDescription ?? $lbTitle) . ", $numEntries entries - $gameTitle for $consoleName";
```

Reply:
In latest I've set this to:

```php
$openGraphDescription = ($lbDescription ?? $lbTitle) . ", $numEntries entries.";
```

I omitted the game title and console name so the description isn't duplicative with the title.